rorapi/settings.py

```diff
@@ -171,8 +171,13 @@
 #     'URL': 'https://digitalscience.figshare.com/ndownloader/files/14399291'
 # }
 
+# GRID = {
+#    'VERSION': '2019-05-06',
+#    'URL': 'https://digitalscience.figshare.com/ndownloader/files/15167609'
+#}
+
 GRID = {
-    'VERSION': '2019-05-06',
+    'VERSION': '2019-09-19',
```

Comment:
This version was in fact version of the latest GRID release, which we do not control (https://grid.ac/downloads). Unfortunately, the "version" of ROR dump is currently bound to the GRID release version. It is also present in the location path of the ROR dump.

It would be good to change this and introduce a ROR dump version, completely separate from GRID version. It could be simply the date when ROR dump was generated. It would require some changes in settings.py and in the location of the dump files.

I think it would be good to do these changes now, as this PR is a good use case for that.

Reply:
I've introduced a ROR dump version. Could you please have a look.